install

```diff
@@ -628,6 +628,7 @@ link_python_app libretime-playout-notify
 
 info "creating libretime-playout working directory"
 mkdir_and_chown "$LIBRETIME_USER" "$WORKING_DIR/playout"
+mkdir_and_chown "$LIBRETIME_USER" "$WORKING_DIR/playout/hls"
```

Comment:
I prefer to have playout create the "$WORKING_DIR/playout/hls" directory, the "$WORKING_DIR/playout" is writable by libretime.

Reply:
the hls directory must be created before nginx starts so we can't let playout create this directory...no?

Reply:
Why ? We only read in this directory during a request, and response with 404 if no files are found. Sound doable to me.

Reply:
if you think it's safe to launch nginx and serve a not-yet-created directory, so i'm good with removing this line